mandatory/2-function-creation.js

```diff
@@ -87,5 +110,5 @@ test("tidyUpString function works", () => {
     "ashleigh",
     "alastair",
     "anne marie",
-  ]); 
+  ]);
 });
```

Comment:
hi Patou, I am not sure how some of these worked, but what I notice is your variables -  removeSpace; removeForwardSlash; returnLowerCase; do not connect to the array you are looping through so their values sort of stay the same.I see u did get an error with this one so this could be why.